package/gluon-client-bridge/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-client-bridge
+PKG_VERSION:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(GLUONDIR)/include/package.mk
+
+define Package/gluon-client-bridge
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  TITLE:=Provides a bridge and a wireless interface for clients to connect to
+  DEPENDS:=+gluon-core +gluon-radio-config
```

Comment:
Can we somehow loosen the dependency on gluon-radio-config? Optimally, it should be possible to create a firmware containing gluon-client-bridge and maybe even gluon-batman-adv-* without gluon-radio-config if no WLAN support is needed.